docs/content/users/install/ddev-installation.md

```diff
@@ -193,7 +222,7 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
 
     ## Traditional Windows
 
-    DDEV does work fine on the Windows side, although it’s quite a bit slower than WSL2 by default. Good results have been reported by users who enabled Mutagen.
+    DDEV does work fine on the Windows side, although it’s quite a bit slower than WSL2 by default. You'll want to enable Mutagen or NFS for tolerable performance. Most users find that Mutagen is the most satisfactory technique. See [Performance](performance.md) for more information, but `ddev config global --mutagen-enabled` is the command to enable mutagen.
```

Comment:
Get to the point with better links:

```suggestion
    DDEV works fine on the Windows side, but it’s slower than WSL2 by default. Enable either [Mutagen](https://ddev.readthedocs.io/en/stable/users/install/performance/#system-requirements) or [NFS](https://ddev.readthedocs.io/en/stable/users/install/performance/#nfs) for the best performance.
```

In this case, the explanation gets in the way of the instruction:

1. The performance page (and lots of other blurbs) heavily emphasize Mutagen over NFS, so there’s no need to offer commentary here on what most users find best. (We’re leading with that recommendation anyway.)
2. The Mutagen link gets right to the point with that command to enable, so there’s no need to be redundant with it here. (It’s also strangely prescriptive to recommend Mutagen or NFS but only offer the command for Mutagen—might as well link thoughtfully and encourage the reader to either pick the quick + obvious answer or read further and make the choice they’re happiest with. No need to broadly link to the performance page and short-circuit their trip in one breath while distracting from further Windows install notes.)

Reply:
Links should always be relative in the docs, not link out to readthedocs.

```suggestion
    DDEV works fine on the Windows side, but it’s slower than WSL2 by default. Enable either [Mutagen](performance/#system-requirements) or [NFS](performance/#nfs) for the best performance.```

Reply:
Yikes, I don’t know what came over me. Thank you for fixing!